packages/playwright-core/src/server/registry/index.ts

```diff
@@ -400,6 +400,17 @@ function readDescriptors(browsersJSON: BrowsersJSON): BrowsersJSONDescriptor[] {
     };
     return descriptor;
   });
+  // Note: chromium-headless-shell has the same revision as chromium.
+  const chromium = descriptors.find(d => d.name === 'chromium')!;
```

Comment:
Perhaps we should actually put this into `browsers.json` then and update `roll_browser.js` script to roll it together with chromium?